packages/react/src/drafts/TabPanels/TabPanels.docs.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "tab_panels",
+  "name": "TabPanels",
+  "status": "draft",
+  "a11yReviewed": false,
+  "stories": [],
+  "props": [
+    {
+      "name": "aria-label",
+      "type": "string",
+      "description": "Used to set the `aria-label` on the `role=\"tablist\"` element."
+    },
+    {
+      "name": "children",
+      "type": "React.ReactNode",
+      "description": "The content of the component, can contain Tabs, Panels but also content before and after Tabs and after the Panels."
+    },
+    {
+      "name": "id",
```

Comment:
Could we make the `id` prop optional here and use the `useId` hook to optionally generate one / use an `id` if it's provided?

Reply:
_done_